deploy/kubernetes/overlays/prow-gke-staging/controller_images.yaml

```diff
@@ -0,0 +1,14 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-provisioner
+          image: gcr.io/gke-release-staging/csi-provisioner:v0.4.1-gke.0
+        - name: csi-attacher
+          image: gcr.io/gke-release-staging/csi-attacher:v0.4.1-gke.0
+        - name: gce-pd-driver
+          image: gcr.io/gke-release-staging/gcp-compute-persistent-disk-csi-driver:v0.2.0-gke.0
```

Comment:
is the gke patch necessary/automatic?

Reply:
also, are we going to add a test for the latest prod version as well?

Reply:
right now it's manual. every time we want to qualify a combination of sidecars and containers, we need to update this config with the versions of each container we want to use.

eventually i hope to make it automatic similar to how it works in kubernetes, where there's a file that stores the latest tagged versions.

I will update the base (stable) config with the prod images once we qualify them and make them available.